### ISSUE TITLE
Domain donation: rgb.network

### ISSUE THREAD

**Issue by rikur** [rgb-org/spec#54]
Grabbed it just in case you guys need a place to call home. Will transfer over free of charge.

**Comment by afilini** [comment]
Sorry for the reply, we are trying to catch up the work now after the BoB!

Thank you very much, I feel so dumb for not having thought about it before. 

Can we at least pay you back the registration fees? Even via LN if you want 😏 😏

**Comment by rikur** [comment]
It's alright, you're doing valuable work and it was an easy way for me to contribute. I went to my registrar (name.com) and saw this:

```
Before starting your domain transfer...
Verify that your domain has been registered for at least 60 days
```

So for now, you can let me know any records I should create. After ~2018-09-04 I can initiate the domain transfer.

**Comment by rikur** [comment]
I think I could transfer it to another name.com account though. Let me know :)

**Comment by afilini** [comment]
Okay, I quickly setup a GitHub page website: could you please point the domain to these ip addresses?

```
185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153
```

Thanks!

**Comment by rikur** [comment]
Done! Will get back to you when the domain can be transferred. Also, feel free to ping in this thread if you need anything else.

**Comment by afilini** [comment]
I opened another issue in the website repo, since this is more of a website task than a specs one.

Further comments will be posted there!
